server/utils/spellcheck.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import (
+	"github.com/agext/levenshtein"
+)
+
+// Spellcheck calculates "The Levenshtein Distance" between two strings which
+// represents the minimum total cost of edits that would convert the first string
+// into the second. If the distance is less than 3, the word is considered misspelled.
+func Spellcheck(given string, suggestion string) bool {
```

Comment:
How about to rename `HasSimilarWord` and so on instead `Spellcheck`? It become more explicit to return bool.

Reply:
renamed to `IsSimilarWord`